### ISSUE TITLE
Display report status changes and their motivations on report page.

### ISSUE THREAD

**Issue by evilaliv3** [globaleaks/whistleblowing-software#4104]
### Proposal

This ticket is dedicated to the analysis and implementation for a functionality to show information about statuses changes and their motivations on the report page.

The idea is to extend the "Comment" section adding a comment for each report  showing the following information
Author: The user that reported
Date: The date of the operation
Text: "Status: $StatusBeforeChange ->  $StatusAfterChange

Examples:
```
Giorgio Fraschini
Date: June 12, 2024
Status: New -> Open
```

```
Giorgio Fraschini
Date: June 12, 2024
Status: Open -> Close
Motivation: Case handled with complete whistleblower's protection
```

**Comment by evilaliv3** [comment]
Cool @msmannan02 

I think the implementation is almost complete and i'm going to open a pull request to perform a review and track the merge.

**Comment by evilaliv3** [comment]
Closing the ticket since the feature is implemented and released in up-to-date version 5.0.12

![image](https://github.com/user-attachments/assets/dd19223e-bb93-47a9-934b-46a1c3d1ef34)
